src/Dhadgar.Nodes/README.md

```
@@ -1,245 +1,117 @@
 
-Or with hot reload for development:
-
-```bash
+# Or with hot reload
 dotnet watch --project src/Dhadgar.Nodes
 ```
 
-The service starts on **http://localhost:5004** (configured in `launchSettings.json`).
-
-### Step 4: Verify the Service
+The service starts on **http://localhost:5040**.
```

markdownlint-cli2 (0.20.0)

Bare URL used

(MD034, no-bare-urls)